### PR TITLE
DOC-2056: initial language/style guide edit

### PR DESCRIPTION
**Pull request by unitCCK**

### CODE REVIEW

File: docs/UnitySimulationHelpInformation.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-* Run name is used by Unity Simulation to label the run being executed on a node  
-* Path to build zip is used for location of the linux player build of the target project, this will be unzipped and ran on a Unity Simulation node
-* Scale factor range is a graph mapping the scale factor values from 0-1 range
-* Scale factor steps is the number of samples between 0-1 that we will simulate on
-* Another way to think of scale factor range and steps is a change in the graph that min and max values in the graph range
+* Run Name is used by Unity Simulation to label the run being executed on a node  
+* Path to build zip is used for the location of the Linux player build of the target project: the user unzips this and runs it on a Unity Simulation node
+* Scale Factor Range is a graph mapping the scale factor values in the 0-1 range
+* Scale Factor Steps is the number of samples between 0-1 that the user simulates on
+* Another way to think of Scale Factor Range and Scale Factor Steps is a change in the graph that min and max values in the graph range
```
Comment (unitCCK) [inline docs/UnitySimulationHelpInformation.md:15]: I'm not sure what this bullet point means. Can you clarify?

File: docs/AnnotatedDataset.md [hunk 2]
```diff
@@ -1,37 +1,44 @@
-# Annotated Dataset 
+# Annotated dataset
```
Comment (unitCCK) [inline docs/AnnotatedDataset.md:1]: I've tried to make it more clear who is doing what in this file, for example are we talking about the authors of the paper or the Synth Det team. This is to align with this guideline in the docs style guide: https://unity-docs.gitbook.io/style-guide/getting-started/writing-tips#use-active-voice 
Let me know if I haven't attributed things correctly, and also if it's OK to refer to the Synth Det team, as I've done a few times.

File: docs/UnitySimulationHelpInformation.md (hunk 1 above)
Comment (JonathanHUnity) [reply docs/UnitySimulationHelpInformation.md:15]: @Saurav-D How about this:
>Scale Factor Steps is the number of scale factors that will be executed in this run. SynthDet will generate hundreds of images for each step, so this number also controls the amount of images generated in the dataset.

**Review by JonathanHUnity — APPROVED** [review]
Looks great. Much more clean!

File: docs/UnitySimulationHelpInformation.md (hunk 1 above)
Comment (unitCCK) [reply docs/UnitySimulationHelpInformation.md]: I've added this text in the latest commit.

**Review by JonathanHUnity — APPROVED** [review]
Looks great. What a difference a professional editor makes!

**Pull request merged**